eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -418,6 +418,34 @@ object Graph {
 
     }
 
+    case class IgnoredEdges(ignoredLocalEdges: Set[ShortChannelId], ignoredRemoteEdges: Set[(PublicKey, PublicKey)]) {
```

Comment:
Can you document this class? It's worth explaining why instead of ignoring a single channel, we ignore all channels between the two nodes (because we assume they're using non-strict forwarding, so none of their channels would work).